src/libcore/slice/mod.rs

```diff
@@ -2466,6 +2466,256 @@ impl<T> [T] {
     {
         self.is_sorted_by(|a, b| f(a).partial_cmp(&f(b)))
     }
+
+    /// Empty out a slice and return a pointer to its contents.
+    ///
+    /// This is equivalent to `mem::replace(&mut slice, &[])`.
+    #[inline]
+    #[unstable(feature = "slice_take", issue = "62280")]
+    fn take<'a>(slice: &mut &'a Self) -> &'a Self {
+        crate::mem::replace(slice, &[])
+    }
+
+    /// Empty out a mutable slice and return a pointer to its contents.
+    ///
+    /// This is equivalent to `mem::replace(&mut slice, &mut [])`.
```

Comment:
```suggestion
    /// This is equivalent to `mem::replace(&mut slice)`.
```

Reply:
^take.  Also note that it's redundant and overwhelming for the author when reviewers go through and write the same comment.

Reply:
> Also, it's redundant and overwhelming for the author when reviewers go through and write the same comment.

The point of these suggestions is that they can be auto-applied and so you don't need to check out your branch and whatnot.

Reply:
FWIW, I have never once autoapplied these suggestions-- I don't know any way to do that that doesn't create an extra commit for each one.

Reply:
There's a button "Add suggestion to batch" to the right?

Reply:
It's grayed out for me :woman_shrugging:

Reply:
@cramertj Try going to the "Files changed" tab. They are greyed out for me if I visit https://github.com/rust-lang/rust/pull/62282 but not if I visit https://github.com/rust-lang/rust/pull/62282/files.